app/templates/people/person/positions/minister/edit.hbs

```diff
@@ -0,0 +1,250 @@
+                        <ProvinceSelect
+                        @selected={{this.model.minister.contacts.firstObject.contactAddress.province}}
+                        @onChange={{fn
+                            (mut
+                            this.model.minister.contacts.firstObject.contactAddress.province
+                            )
+                        }}
+                        />
+                    </:content>
+                    </Item>
+                </:left>
+                <:right as |Item|>
+                    <Item @labelFor="contact-email">
+                    <:label>E-mail</:label>
+                    <:content>
+                        <AuInput
+                        @width="block"
+                        @value={{this.model.minister.contacts.firstObject.email}}
+                        id="contact-email"
+                        />
+                    </:content>
+                    </Item>
+                    <Item @labelFor="contact-telephone">
+                    <:label>Telefoonnummer</:label>
+                    <:content>
+                        <AuInput
+                        @width="block"
+                        @value={{this.model.minister.contacts.firstObject.telephone}}
+                        id="contact-telephone"
+                        />
+                    </:content>
+                    </Item>
+                </:right>
+                </Card.Columns>
+            </:card>
+            </EditCard>
+
+            <EditCard>
+                <:title>Aanstelling</:title>
+                <:card as |Card|>
+                <Card.Columns>
+                    <:left as |Item|>
+                    <Item @labelFor="start-date">
+                        <:label>Startdatum</:label>
+                        <:content>
+                        <Datepicker
+                            @id="start-date"
+                            @onChange={{fn (mut this.model.minister.agentStartDate)}}
+                            @value={{this.model.minister.agentStartDate}}
+                        />
+                        </:content>
+                    </Item>
+                    </:left>
+                    <:right as |Item|>
+                    <Item @labelFor="end-date">
+                        <:label>Einddatum</:label>
+                        <:content>
+                        <Datepicker
+                            @disabled={{this.isCurrentPosition}}
+                            @id="end-date"
+                            @onChange={{fn (mut this.model.minister.agentEndDate)}}
+                            @value={{this.model.minister.agentEndDate}}
+                        />
+                        </:content>
+                    </Item>
+                    <Item @labelFor="end-date">
+                        <:label></:label>
+                        <:content>
+                        <AuControlCheckbox
+                            @checked={{this.isCurrentPosition}}
```

Comment:
`@checked` is what causes the value to be 2-way-bound and is what actually updates the value at the moment.

I would change it to just a `checked` attribute instead and do the state updates in the action. That makes it more clear how everything works when reading the code.